test/container_test.py

```diff
@@ -1309,6 +1314,39 @@ async def custom_heartbeater(self):
     assert "Traceback" not in caplog.text  # should not print a full traceback - don't scare users!
 
 
+@skip_github_non_linux
+@pytest.mark.usefixtures("server_url_env")
+def test_sigint_termination_input_concurrent(servicer):
```

Comment:
Kind of tests the same thing as `test_cancellation_stops_task_with_concurrent_inputs` but for completeness we shouldn't assume that cancellations use sigint